crates/rslint_lexer/src/lib.rs

```diff
@@ -745,6 +758,8 @@ impl<'src> Lexer<'src> {
         None
     }
 
+    // Read a number which does not start with 0, since that can be more things and is handled
+    // by another function
```

Comment:
```suggestion
    // Read a number that doesn't start with 0, since that can be more things and is handled
    // by another function
```

I am having hard time to understand `since that can be more things and is handled by another function`. Would it be possible to reword it? Can we point to which function we are talking about?

Reply:
You can ignore it. This is an old comment. It was removed already.